### ISSUE TITLE
[Bug] [1.10.59] BlockchainProcessor encountered an exception.

### ISSUE THREAD

**Issue by Texnomic** [NethermindEth/nethermind#2988]
When I increase the `MemoryHint` to `61440000000`, I get the following exception.

This issue might be related to #2862

```csharp
System.InvalidOperationException: The LinkedList is empty.
   at System.Collections.Generic.LinkedList`1.RemoveFirst()
   at Nethermind.Core.Caching.LruCache`2.Replace(TKey key, TValue value)
   at Nethermind.Core.Caching.LruCache`2.Set(TKey key, TValue val)
   at Nethermind.Trie.CachingStore.get_Item(Byte[] key)
   at Nethermind.Trie.Pruning.TrieStore.LoadRlp(Keccak keccak, IKeyValueStore keyValueStore)
   at Nethermind.Trie.Pruning.TrieStore.LoadRlp(Keccak keccak)
   at Nethermind.Trie.TrieNode.ResolveNode(ITrieNodeResolver tree)
   at Nethermind.Trie.PatriciaTree.Run(Span`1 updatePath, Int32 nibblesCount, Byte[] updateValue, Boolean isUpdate, Boolean ignoreMissingDelete, Keccak startRootHash)
   at Nethermind.Trie.PatriciaTree.Get(Span`1 rawKey, Keccak rootHash)
   at Nethermind.State.StateTree.Get(Address address, Keccak rootHash)
   at Nethermind.State.StateProvider.GetState(Address address)
   at Nethermind.State.StateProvider.GetAndAddToCache(Address address)
   at Nethermind.State.StateProvider.AccountExists(Address address)
   at Nethermind.Evm.TransactionProcessor.Execute(Transaction transaction, BlockHeader block, ITxTracer txTracer, Boolean isCall)
   at Nethermind.Evm.TransactionProcessor.Execute(Transaction transaction, BlockHeader block, ITxTracer txTracer)
   at Nethermind.Blockchain.Processing.BlockProcessor.ProcessTransactions(Block block, ProcessingOptions processingOptions, IBlockTracer blockTracer)
   at Nethermind.Blockchain.Processing.BlockProcessor.ProcessBlock(Block block, IBlockTracer blockTracer, ProcessingOptions options)
   at Nethermind.Blockchain.Processing.BlockProcessor.ProcessOne(Block suggestedBlock, ProcessingOptions options, IBlockTracer blockTracer)
   at Nethermind.Blockchain.Processing.BlockProcessor.Process(Keccak newBranchStateRoot, List`1 suggestedBlocks, ProcessingOptions options, IBlockTracer blockTracer)
   at Nethermind.Blockchain.Processing.BlockchainProcessor.ProcessBranch(ProcessingBranch processingBranch, ProcessingOptions options, IBlockTracer tracer)
   at Nethermind.Blockchain.Processing.BlockchainProcessor.Process(Block suggestedBlock, ProcessingOptions options, IBlockTracer tracer)
   at Nethermind.Blockchain.Processing.BlockchainProcessor.RunProcessingLoop()
   at System.Threading.Tasks.Task.InnerInvoke()
   at System.Threading.Tasks.Task.<>c.<.cctor>b__277_0(Object obj)
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location ---
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
```

**Comment by tkstanczak** [comment]
Hi @Texnomic , thank you, let me have a look at the code for this.

**Comment by tkstanczak** [comment]
Fixed in 1.10.73, sorry @Texnomic it took so long
